packages/bot-web-ui/src/stores/dashboard-store.ts

```diff
@@ -1,6 +1,8 @@
 import { action, computed, makeObservable, observable, reaction } from 'mobx';
 import { setColors } from '@deriv/bot-skeleton';
 import { TStores } from '@deriv/stores/types';
+import { botNotification } from 'Components/bot-notification/bot-notification';
+import { notification_message, NOTIFICATION_TYPE } from 'Components/bot-notification/bot-notification-utils';
```

Comment:
Can we use type in the import here?

```suggestion
import { notification_message, type NOTIFICATION_TYPE } from 'Components/bot-notification/bot-notification-utils';
```